Makefile

```diff
@@ -101,7 +104,12 @@ USAGEFILES = docs/usage.md
 usage: install
 	$(foreach md,$(USAGEFILES),$(USAGE_CMD)$(nl))
 
-.PHONY: check-evy-fmt doc doctest evy-fmt lint toc
+GODOC_CMD = ./scripts/gengodoc.awk  -v cmd="evy --help" $(md) > $(O)/out.md && mv $(O)/out.md $(md)
+GODOCFILES = main.go
+godoc: install
+	$(foreach md,$(GODOCFILES),$(GODOC_CMD)$(nl))
```

Comment:
This foreach doesn't really make sense as if you add more files to `GODOCFILES` it will still run `evy --help` as that is the command in `GODOC_CMD`. I'd suggest simplifying this, getting rid of the two vars and just doing:

```
godoc: install
        ./scripts/gengodoc.awk -v cmd="evy --help" main.go > $(O)/out.go && mv $(O)/out.go > main.go
```

(note: i just typed this in the browser and didn't check/test for typos)

I also note that GODOC_CMD calls the tmp file `.md` even though it is go. doesn't matter much though.


but but but, just added a comment below about extracting the command from the usage message so you don't need `-v cmd=...` here, which means you could keep this loop. I think that may be better.